a11y/views/testSpec.scala

```diff
@@ -203,32 +205,21 @@ class testSpec extends A11ySpec {
 
     server.stubFor(
       put(urlMatching(s"/keystore/pertax-frontend/.*"))
```

Comment:
is keystore still used? I think only http-caching was using keystore.

Reply:
It was used only in test stubs. Removed it.